src/os/env.go

```diff
@@ -137,5 +138,13 @@ func Clearenv() {
 // Environ returns a copy of strings representing the environment,
 // in the form "key=value".
 func Environ() []string {
-	return syscall.Environ()
+	orig := syscall.Environ()
+	single := strings.Join(orig, "")
```

Comment:
Upsteam does the copy in syscall.Environ(), we might want to do the same in case someone calls that directly?